un/setenv are protected by __STRICT_ANSI__

--- origsrc/kdevplatform-1.1.1/CMakeLists.txt	2010-11-25 15:07:24.000000000 -0600
+++ src/kdevplatform-1.1.1/CMakeLists.txt	2010-12-31 01:42:12.759953400 -0600
@@ -51,7 +51,7 @@ if(CMAKE_COMPILER_IS_GNUCXX)
   macro_ensure_version("4.3.0" "${_gcc_version}" GCC_IS_NEWER_THAN_4_3)
   if (GCC_IS_NEWER_THAN_4_3)
     message(STATUS "Enabling c++0x support for unordered map")
-    add_definitions( -std=c++0x ) # For unordered_map
+    add_definitions( -std=gnu++0x ) # For unordered_map
   else(GCC_IS_NEWER_THAN_4_3)
     add_definitions( -DGXX_LT_4_3 )
   endif (GCC_IS_NEWER_THAN_4_3)
