--- origsrc/kdevplatform-1.5.1/plugins/dashboard/declarative/CMakeLists.txt	2013-05-29 15:51:33.000000000 -0500
+++ src/kdevplatform-1.5.1/plugins/dashboard/declarative/CMakeLists.txt	2013-08-30 18:38:38.350195600 -0500
@@ -2,7 +2,7 @@ project(kdevelop-declarative)
 
 include_directories( ${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR} )
 
-kde4_add_library(kdevelopdashboarddeclarativeplugin SHARED kdevelopdashboarddeclarativeplugin.cpp icoreobject.cpp)
+kde4_add_library(kdevelopdashboarddeclarativeplugin MODULE kdevelopdashboarddeclarativeplugin.cpp icoreobject.cpp)
 
 target_link_libraries(kdevelopdashboarddeclarativeplugin
     ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTDECLARATIVE_LIBRARY}
