--- origsrc/kdevplatform-0.9.90/language/util/setrepository.cpp	2009-01-27 16:46:01.000000000 -0600
+++ src/kdevplatform-0.9.90/language/util/setrepository.cpp	2009-01-28 02:37:32.125500000 -0600
@@ -476,7 +476,7 @@ struct Set::Iterator::IteratorPrivate {
       
       if(node->contiguous())
         break; //We need no finer granularity, because the range is contiguous
-      node = repository->dataRepository.itemFromIndex(node->leftNode);
+      node = repository->getDataRepository().itemFromIndex(node->leftNode);
     } while(node);
     Q_ASSERT(currentIndex >= nodeStack[0]->start);
   }
